### PR TITLE
Placate a bunch of Allman style violation for #8805

### PR DESCRIPTION
**Pull request by thewilsonator**
I don't necessarily agree with all of these but they show up as conflict for the other PR.

This is a subset, I'll do the other ones later.

### CODE REVIEW

**Review by jmdavis — APPROVED** [review]

File: std/base64.d [hunk 1]
```diff
@@ -2060,7 +2060,8 @@ class Base64Exception : Exception
     // Verify that any existing workaround that uses & still works.
     InputRange ir2;
     OutputRange or2;
-    () @trusted {
+    () @trusted
+    {
```
Comment (thewilsonator) [inline std/base64.d:2064]: specifically I don't like this type of change

File: std/conv.d [hunk 2]
```diff
@@ -1208,7 +1209,8 @@ if (!(is(S : T) &&
 */
 private template isSwitchable(E)
 {
-    enum bool isSwitchable = is(typeof({
```
Comment (thewilsonator) [inline std/conv.d:1211]: and I really don't like this type of change

**Comment by CyberShadow** [comment]
Can we fix Dscanner instead?

**Comment by thewilsonator** [comment]
@WebFreak001 ?

File: std/base64.d (hunk 1 above)
Comment (WebFreak001) [reply std/base64.d:2064]: this definitely looks more correct for allman style, would not remove this from D-Scanner but possibly add some kind of config for this

File: std/conv.d (hunk 2 above)
Comment (WebFreak001) [reply std/conv.d:1211]: yeah this looks like a bug in D-Scanner

File: std/base64.d (hunk 1 above)
Comment (thewilsonator) [reply std/base64.d:2064]: That would be great, thanks!

**Comment by RazvanN7** [comment]
any update on this?

**Comment by WebFreak001** [comment]
ah none yet, I forgot about this PR

**Comment by thewilsonator** [comment]
ping

**Comment by maxhaton** [comment]
> Can we fix Dscanner instead?

how about stop using it, even.

Enforce trivial matters of style with a formatter, not a linter.

**Pull request closed without merging**